src/utils/FalsyValuesUtils.ts

```diff
@@ -4,3 +4,9 @@ export const convertUndefinedAndNullToEmptyString = (value: any) =>
     _.isUndefined(value) || _.isNull(value)
         ? ''
         : value;
+
+export const callIfDefined = (callback: any, ...args: any[]) => {
+    if (typeof callback === typeof Function) {

```

Comment:
can't you just do typeof callback === 'function' ?

Reply:
yes, but I prefer to avoid string literals